odk1-src/central-intro.rst

```diff
@@ -44,15 +44,19 @@ See `What is coming in Central <https://forum.getodk.org/t/whats-coming-in-centr
 
-Additionally, Central does not support every feature that Aggregate does. There are some Aggregate features that we do not intend to ever support in Central, especially around data preview and visualization. You can find a list of these differences and some of the planned improvements to Central `here <https://forum.getodk.org/t/whats-coming-in-central-over-the-next-few-years/19677>`_.
+Please note that Central does not support every feature that Aggregate does. There are some Aggregate features that we do not intend to ever support in Central, especially around data preview and visualization. You can find a list of these differences and some of the planned improvements to Central `here <https://forum.getodk.org/t/whats-coming-in-central-over-the-next-few-years/19677>`_.
 
-On the other hand, Central supports some things that Aggregate does not. Projects let you partition your server into different sandboxes to support multiple independent teams. Managed encryption makes the process of handling encrypted form data significantly easier and in many cases more securely. Central allows direct upload of XLSForm files. It also offers an OData API to easily synchronize your live form data to desktop visualization and dashboard tools.
+On the other hand, Central supports some things that Aggregate does not. For example:
 
-If you are an adventurous user who is comfortable with (new) technology, you are happy with the supported features, and you are okay with the risks with using early release software, please consider giving Central a try and giving us your `feedback <https://forum.getodk.org/c/features>`_.
+- :ref:`direct upload of XLSForm files <central-forms-upload>` makes form management easier
+- :doc:`projects <central-projects>` let you partition your server into different sandboxes to support multiple independent teams
+- :ref:`the OData API <central-submissions-odata>` makes it easy to synchronize your live form data to desktop visualization and dashboard tools
+- :doc:`managed encryption <central-encryption>` makes the process of handling encrypted form data significantly easier and in many cases more secure
 
-If you finished reading all the above and you're not feeling too sure about it, we suggest sitting it out for a little while longer. Keep watching the `release announcements board <https://forum.getodk.org/c/releases>`_ for future updates, and we'll be sure to sound the bells when we're sure things are ready for a broader audience.
+
+If you are happy with the supported features, please give Central a try and provide your `feedback <https://forum.getodk.org/c/features>`_.
```

Comment:
s/If you are happy with the supported features//

https://forum.getodk.org/c/support instead of features?